src/runners/baseRunner/BaseRunnerResultAnalyzer.ts

```diff
@@ -41,35 +42,61 @@ export abstract class BaseRunnerResultAnalyzer {
 
     public feedBack(): ITestResult[] {
         const result: ITestResult[] = [];
+        const itemMap: Map<string, ITestItem> = new Map<string, ITestItem>();
         for (const test of this.tests) {
-            this.processTestItemRecursively(test, result);
+            this.flatTestItems(test, itemMap);
         }
+        this.parseResults(result, itemMap);
+        this.addSkippedTests(result, itemMap);
         return result;
     }
 
-    protected processTestItemRecursively(testItem: ITestItem, resultList: ITestResult[]): void {
-        if (testItem.level === TestLevel.Method) {
-            resultList.push(this.processMethod(testItem));
-        } else {
-            testItem.children.forEach((child: ITestItem) => this.processTestItemRecursively(child, resultList));
+    protected abstract processData(data: string): void;
+
+    protected flatTestItems(item: ITestItem, map: Map<string, ITestItem>): void {
+        if (item.level === TestLevel.Method) {
+            map.set(item.fullName, item);
+        } else if (item.children) {
+            item.children.forEach((child: ITestItem) => this.flatTestItems(child, map));
         }
     }
 
-    protected abstract processData(data: string): void;
+    protected parseResults(resultArray: ITestResult[], itemMap: Map<string, ITestItem>): void {
+        for (const [key, value] of this.testResults.entries()) {
+            let result: ITestResult = Object.assign(_.cloneDeep(defaultResult), {
```

Comment:
Why use clone**Deep** here?

Reply:
Will change to `Object.assign({}, defaultResult, { ... })`